modules/apps/account/account-api/src/main/java/com/liferay/account/service/AccountGroupRelLocalService.java

```diff
@@ -75,13 +75,29 @@
 	@Indexable(type = IndexableType.REINDEX)
 	public AccountGroupRel addAccountGroupRel(AccountGroupRel accountGroupRel);
 
+	/**
+	 * @deprecated As of Cavanaugh (7.4.x)
```

Comment:
I think we should also include a "replaced by" comment with a link. There's a lot of examples in other services you can reference.